src/tlo/methods/malaria.py

```diff
@@ -755,6 +755,16 @@ def report_daly_values(self):
 
         return health_values.loc[df.is_alive]  # returns the series
 
+
+    def report_prevalence(self):
```

Comment:
Thanks for changing this

Reply:
Just to note also, if reporting point prevalence each month, this could potentially miss some cases of malaria which occur and then resolve within the month. One other option could be to use the property ma_date_symptoms to find all malaria cases who have had onset of symptoms within the last time period. Whichever way you prefer is ok.

Reply:
Oh interesting! I suppose most of the other modules are also point prevalence, so maybe for consistency with that, we keep it this way. But if having more of a period prevalence is more useful to you/in general, happy to change it!